backend/backend/config.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     OLLAMA_ENDPOINT: str = "http://ollama:7869"
-    OLLAMA_MODEL: str = "llama3.2"
+    OLLAMA_MODEL: str = "llama3.1:8b"
```

Comment:
Consider using a more specific version of the LLaMA model, as 'llama3.1:8b' may not be the latest or most suitable version for your application.